mrp_bom_component_menu/views/mrp_bom_component_view.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2017 ForgeFlow
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="mrp_bom_line_tree_view" model="ir.ui.view">
+        <field name="name">mrp.bom.line.tree.view</field>
+        <field name="model">mrp.bom.line</field>
+        <field name="arch" type="xml">
+            <tree create="0">
```

Comment:
```suggestion
            <tree create="0" delete="0">
```
I guess this can be an improvement to the module. Instead of just not enabling the creation, also prevent the removal.